pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -33,10 +33,12 @@ import (
 	meshconfig "istio.io/api/mesh/v1alpha1"
 	mpb "istio.io/api/mixer/v1"
 	mccpb "istio.io/api/mixer/v1/config/client"
+
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pilot/pkg/networking/plugin"
 	"istio.io/istio/pilot/pkg/networking/util"
-	"istio.io/istio/pkg/config"
+	"istio.io/istio/pkg/config/host"
+
```

Comment:
nit: why add a space between these two lines?

Reply:
my auto-formatter is grouping together imports from "this" project. "pkg" is an external repo, so it separated it out.

Reply:
i'm very confused by the import groupings. i've lost track of our current style recommendations, but originally we had 3 groups: built-in, third-party, istio.

here, we have *5* groupings: built-in, third-party, istio/api, istio/istio, and istio/pkg.

i don't know what the right thing to do here is, but it feels like we should maybe (at least) collapse istio/api and istio/pkg blocks... and then maybe either merge that into a single istio block or the third-party block.

but i also don't want to spend too much time fighting auto-formatters.

Reply:
Yeah I feel your pain ... I'm not sure we actually have any particular style WRT groupings.  I actually just had this conversation with Kuat recently.  Especially now that we've gone to multi-repo, I'm not sure what the right thing to do is.  For example, do other istio repos warrant their own grouping, or should they just be bundled with "everything not in this project"?

We should probably discuss in some WG (test/release?)

Reply:
maybe we just need to pass `-local istio.io` instead of `-local Istio.io/istio` to the formatter? not sure if that is legal though